packages/utils/src/ts-eslint/Rule.ts

```diff
@@ -423,7 +423,6 @@ interface RuleListenerBaseSelectors {
   AssignmentExpression?: RuleFunction<TSESTree.AssignmentExpression>;
   AssignmentPattern?: RuleFunction<TSESTree.AssignmentPattern>;
   AwaitExpression?: RuleFunction<TSESTree.AwaitExpression>;
-  BigIntLiteral?: RuleFunction<TSESTree.BigIntLiteral>;
```

Comment:
There is no such node as `BigIntLiteral`, it is just an extension of `LiteralBase` which have `type: AST_NODE_TYPES.Literal`.